multimodal/setup.py

```diff
@@ -71,10 +71,13 @@
 extras_require = {
     "PyMuPDF": [
         "PyMuPDF<=1.21.1",
-    ]
+    ],
+    "pdf2image": [
```

Comment:
Considering the small size of the pdf2image package, is it possible to include it in the `install_requires` without causing significant churns?